tests/unit/test_merger_arxiv2arxiv.py

```diff
@@ -674,7 +674,6 @@ def test_head_curates_author_no_duplicate():
 
     expected_merged = {
         'authors': [
-            {'full_name': 'Li, Zhengxiang'},
```

Comment:
I understand this, it was removed before already (present in `root` and `update`, not in `head`), so it shoudn't be added back, but then why is there a conflict to remove the remaining author?

Reply:
So the reason of the conflict is that we have `KEEP_UPDATE_ENTITIES_CONFLICT_ON_HEAD_DELETE` for authors and in list unifier the entry is translated to nodes as following:
`[({'full_name': 'Li, Zhengxiang'}, NOTHING, {'full_name': 'Li, Zhengxiang'})]`. Also in list unifier [we do this check](https://github.com/inveniosoftware-contrib/json-merger/blob/69331b1355231082d220ce5f44050ecc9b13fa77/json_merger/list_unify.py#L117-L121) and we add the conflict. Then, because head is Nothing from `_recursive_merge `we get `authors: []` and in fact authors are filled in `postprocess_results` from this conflict.

Reply:
ok, that makes sense.